app/src/main/java/org/mozilla/focus/fragment/BrowserFragment.java

```diff
@@ -86,6 +91,7 @@
     private static final int ANIMATION_DURATION = 300;
     private static final String ARGUMENT_URL = "url";
     private static final String RESTORE_KEY_DOWNLOAD = "download";
+    private static final String SCREENSHOT_PATH = Environment.getExternalStorageDirectory() + File.separator + Environment.DIRECTORY_PICTURES + File.separator + "Screenshots" + File.separator;
```

Comment:
Is there any concern to not use
`Environment.getExternalStoragePublicDirectory(Environment.DIRECTORY_PICTURES) + File.separator + "Screenshots" + File.separator` here?

Reply:
This is just a probably slightly more promising way to get the correct but same directory. It does not matters anymore now anyway if we're not listening to this. 😜